doc/design/data-retention-policy.md

```diff
@@ -0,0 +1,57 @@
+# Data Retention Policy
+
+Dandihub data storage on AWS EFS is expensive, and we suppose that significant portions of the data
+currently stored are no longer used. Data migration is where the cost becomes extreme.
+
+## Persistent Data locations
+
+Each user has access to 2 locations: `/home/{user}` and `/shared/`
+
+Within jupyterhub `/home/{user}` the user always sees `/home/jovyan`, but is stored in EFS as their GitHub
+username.
+
+## Known cache file cleanup 
+
+We should be able to safely remove the following:
+ - `/home/{user}/.cache`
+ - `nwb_cache` 
+ - Yarn Cache
+ - `__pycache__`
+ - pip cache
```

Comment:
In case user is still active -- I think it would be useful to report to the long running users, after reaching some threshold on any of those folders (e.g. 50MB) asking to clean them up.

Reply:
Hi @asmacdo, should we add a separate point here about monitoring and reporting the quotas of cache directories?